substrate/client/cli/src/params/shared_params.rs

```diff
@@ -53,9 +53,14 @@ pub struct SharedParams {
 	pub detailed_log_output: bool,
 
 	/// Disable log color output.
-	#[arg(long)]
+	#[cfg_attr(not(windows), arg(long))]
+	#[cfg_attr(windows, arg(long, default_value_t = true, action = clap::ArgAction::Set))]
 	pub disable_log_color: bool,
 
+	/// Use UTC time in log output.
+	#[arg(long)]
+	pub use_utc_log_time: bool,
```

Comment:
Hm, I'd probably prefer to have this take a parameter, e.g. `--timestamp-format=$format` where $format can be `local` (default) or `iso8601` (since what you've done is not just to force UTC timestamps - you've changed the format of the timestamp to be like iso8601). It's also more forward compatible if we'd want to add other formats, or make the timestamp format fully configurable with an `strftime`-like string.

Reply:
I like that, but ISO 8601 is not just UTC, so I it seems like there should still be a distinction between UTC and non-UTC in logs if you really want to have local time there (I suspect that is current expectation from Substrate users that would be undesirable to change).

Reply:
Well then, it could be `iso8601-utc` I guess?

Reply:
Hm, I read about ISO 8601, what a mess... There are so many way to compose valid ISO 8601 date and time that it is hardly helpful to name an option like that. Technically both local and UTC time in PR description are valid RFC 3339 formats, so the only difference between them is local vs UTC time and `T` vs ` ` between date and time (both valid).

What about something like this?:
* `--timestamp-format=default`
* `--timestamp-format=tracing-subscriber`

`default` is the current default of Substrate, `tracing-subscriber` is format used in that crate. Standard doesn't say anything about number of digits for fractional seconds, so both 3 and 6 is valid according to RFC 3339.

Here is helpful visualization: https://ijmacd.github.io/rfc3339-iso8601/